src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -335,13 +367,21 @@ void putEntries() {
         /*
          *  Key Pair Generator engine
          */
-        putService(new ProviderServiceA(this, "KeyPairGenerator",
-            "EC", "sun.security.ec.ECKeyPairGenerator", ATTRS));
+        /* Disabling OpenSSL usage in AIX due to perfomance regression observed */
+        if (useNativeECKeyGen &&
+            (NativeCrypto.getVersionIfAvailable() >= NativeCrypto.OPENSSL_VERSION_1_1_0)&&
+            (!isAIX)) {
```

Comment:
Put `&&` at the beginning of each continued line, `) {` on a line by itself and remove extra parentheses:
```java
        if (useNativeECKeyGen
            && (NativeCrypto.getVersionIfAvailable() >= NativeCrypto.OPENSSL_VERSION_1_1_0)
            && !isAIX
        ) {
```

Reply:
Done.